src/loaders/express.js

```diff
@@ -49,7 +49,7 @@ export default (app) => {
 
     // Set the API call rate limit only on production
     if (process.env.NODE_ENV === "production") {
-        const api_rate_limiter = new RateLimit({
+        const api_rate_limiter = RateLimit({
```

Comment:
Why this change?

Reply:
It isn't working correctly and crashed the app in production mode. There was probably an update to this library